.github/workflows/terraform-static-analysis.yml

```diff
@@ -1,6 +1,9 @@
 name: Terraform Static Code Analysis
 
 on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '* 7 * * 1-5'
```

Comment:
I read this as every minute past 7am, Mon-Fri. Should this not be: 0 7 * * 1-5

Reply:
🤦 you are 100% correct, I'll get it updated in a few

Reply:
cron expression updated